connectors/src/connectors/salesforce/index.ts

```diff
@@ -177,21 +230,25 @@ export class SalesforceConnectorManager extends BaseConnectorManager<null> {
       );
     }
 
-    // We return a single fake node just to display the message in the UI instead of "No documents found".
-    const node: ContentNode = {
-      internalId: `salesforce-synced-query-root-${this.connectorId}`,
-      parentInternalId: null,
-      type: "folder",
-      title: "Synced Queries are set by Dust - no permissions needed.",
-      sourceUrl: null,
-      expandable: false,
-      preventSelection: false,
-      permission: "read",
-      lastUpdatedAt: null,
-      mimeType: INTERNAL_MIME_TYPES.SALESFORCE.SYNCED_QUERY_FOLDER,
-    };
-
-    return new Ok([node]);
+    const queries =
```

Comment:
That means when they setup the connection we are back to displaying "No documents found".  I guess okay cause we are supposed to activate it ourselves and guide the users but I think it means we should write an internal guide, I know some other workspaces were interested 🙏🏻

Reply:
Also that means when they create the connection they set the credentials and then they are stuck here: 
<img width="814" alt="Screenshot 2025-07-08 at 14 42 47" src="https://github.com/user-attachments/assets/4dd0b8b8-d9b1-4bf4-8bb7-13e76ff6d83c" />

Reply:
Yeah I know not the best but it's also very white glove for now. If we scale the number of workspaces we can have a special handling for that case for Salesforce but I don't think it's required right now?